scipy/signal/_peak_finding.py

```diff
@@ -257,24 +257,39 @@ def peak_prominences(vector, peaks, wlen=None):
         Indices of peaks in `vector`.
     wlen : int, optional
         A window lenght in samples that limits the search for the lowest
-        contour line two a symmetric interval around the evaluated peak. If
-        not given the entire vector is used. Use this parameter to speed up
-        the calculation significantly for large vectors.
+        contour line two a symmetric interval around the evaluated peak. If not
+        given the entire vector is used. Use this parameter to speed up the
+        calculation significantly for large vectors.
+    ret_bases : bool
+        If true, the peaks' left and right bases are returned. See returned
+        arguments below.
 
     Returns
     -------
     prominences : ndarray
         The calculated prominences for each peak in `peaks`.
+    left_bases, right_bases : ndarray, returned if ret_bases is true
```

Comment:
after ndarray a newline is needed for rendering the type correctly.

Reply:
Which type? I got this from a the example in [scipy.integrate.odeint](https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.odeint.html#scipy.integrate.odeint) (returned parameter `infodict`).

Reply:
I think the `odeint` example is less than ideal in this respect